lib/obfuscate_id.rb

```diff
@@ -34,8 +35,12 @@ def has_obfuscated_id?
       true
     end
 
-    def deobfuscate_id(obfuscated_id)
```

Comment:
I changed the naming convention here because the `obfuscate_id` method name was already taken.